frontend/src/SignupForm/SignupForm.js

```diff
@@ -48,10 +78,69 @@ const Span = styled.span`
   justify-content: center;
 `;
 
+const UploadedImg = styled.img`
+  margin: 10px;
+  padding: 10px;
+  display: block;
+`;
+
+const VisibleDiv = styled.div`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: white;
+  border: 1px solid #209cee;
+  border-radius: 10px;
+
+  &:hover {
+    border: 1 px solid #0099ff;
+    background-color: #88caf6;
+    transition: border-color 0.25s ease-in-out 0s,
+      box-shadow 0.1s ease-in-out 0s, background-color 0.25s ease-in-out 0s,
+      color 0.25s ease-in-out 0s;
+  }
+
+  &:active {
+    background-color: #1081cb;
+    font-color: white;
+  }
+
+  &:focus {
+    color: purple;
+  }
+`;
+
 const SignupForm = React.memo(({ processForm }) => {
   const [username, setUsername] = useState("");
   const [password, setPassword] = useState("");
   const [email, setEmail] = useState("");
+  const [photo, setPhoto] = useState(null);
+  const [name, setName] = useState("");
+  const [phoneNumber, setPhoneNumber] = useState("");
+  const [city, setCity] = useState("");
+  const [state, setState] = useState("");
+
+  const handlePhotoSubmit = useCallback((event) => {
+    const reader = new FileReader();
+    const file = event.currentTarget.files[0];
+
+    const TWO_MEGABYTES = 1000 * 1000 * 2;
```

Comment:
Hoist this up above the component. No need to re-declare this constant any time a submission is made.